externs/shaka/player.js

```diff
@@ -97,6 +98,9 @@ shaka.extern.StateChange;
  * @property {number} droppedFrames
  *   The total number of frames dropped by the Player.  This may be
  *   <code>NaN</code> if this is not supported by the browser.
+ * @property {number} corruptedFrames
+ *   The total number of frames corrupted by the Player.  This may be
```

Comment:
My only complaint about this is that it sounds like the Player corrupted them somehow.  Perhaps this should say "number of corrupted frames dropped by the browser".

Reply:
Sorry, it was a mistake! I changed it.